jumanji/training/configs/env/rware.yaml

```
@@ -0,0 +1,29 @@
+name: rware
+registered_version: Rware-v0
+
+network:
+    policy_layers: [64,64]
+    value_layers: [128,128]
+    transformer_num_blocks: 4
+    transformer_num_heads: 8
+    transformer_key_size: 16
+    transformer_mlp_units: [512]
+
+training:
+    num_epochs: 500
+    num_learner_steps_per_epoch: 100
+    n_steps: 20
+    total_batch_size: 128
+
+evaluation:
+    eval_total_batch_size: 5000
+    greedy_eval_total_batch_size: 5000
+
+a2c:
+    normalize_advantage: False
+    discount_factor: 0.99
+    bootstrapping_factor: 0.95
+    l_pg: 1.0
+    l_td: 1.0
+    l_en: 0.01
+    learning_rate: 3e-4
```

3e-4 might be high for attention layers. Unless you get stable learning curves, I would suggest a lower learning rate.
```suggestion
    learning_rate: 1e-4
```

---

Curves seem quite stable to me, so resolving this.